### PR TITLE
Kubevirt: vm template list: create YAML, vm template yaml redirect

### PR DESCRIPTION
**Pull request by yaacov**
Depends on https://github.com/openshift/console/pull/1973

Peek 2019-07-10 11-17

### CODE REVIEW

**Comment by yaacov** [comment]
@mareklibra @vojtechszocs @suomiy @rawagner please review.

Only commits from https://github.com/openshift/console/pull/1974/commits/30690cd3c618a9ccb0a9123b50ebc9d0890c50d4 belong to this PR, rest are from #1973

File: frontend/public/components/edit-yaml.jsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ const stateToProps = ({k8s, UI}) => ({
  * This component loads the entire Ace editor library (~100kB) with it.
  * Consider using `AsyncComponent` to dynamically load this component when needed.
  */
-/** @augments {React.Component<{obj?: any, create: boolean, kind: string, redirectURL?: string}>} */
+/** @augments {React.Component<{obj?: any, create: boolean, kind: string, redirectURL?: string, resourceObjPath?: (obj: k8sKind, objRef: string) => string}>} */
```
[the remark this answers is not in the document]
Comment (suomiy) [reply frontend/public/components/edit-yaml.jsx]: sorry, moved to #1973

File: frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-create-yaml.tsx [hunk 2]
```diff
@@ -1,13 +1,53 @@
...
+  resourcePathFromModel,
+} from '@console/internal/components/utils';
+import { ErrorPage404 } from '@console/internal/components/error';
+import { getNamespace, getName } from '@console/shared';
+
 import { VMTemplateYAMLTemplates } from '../../models/templates';
+import { VM_TEMPLATE_CREATE_HEADER } from '../../constants/vm-templates';
+
+const CreateVMTemplateYAMLConnected = connectToPlural((props: CreateYAMLProps) => {
+  const { match, kindsInFlight, kindObj } = props;
```
[the remark this answers is not in the document]
Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-create-yaml.tsx]: Nice, thanks :+1:

**Comment by yaacov** [comment]
@mareklibra @vojtechszocs @suomiy @rawagner hi,

I forgot to put this in wip until #1973 is merged, but it's merged now ... so removing wip :-).

please review.

**Pull request merged**